src/pages/UnLikedFoodPage.js

```diff
@@ -60,25 +102,37 @@ const UnLikedFoodPageStyled = styled.section`
 `;
 
 const Header = styled.header`
-  background-color: var(--white);
-  padding: 5vmin 1rem;
+  background-color: var(--peach);
   width: 100%;
   position: fixed;
   top: 0;
   z-index: 10;
   overflow: hidden;
   box-shadow: var(--box-shadow-header-drop);
   grid-row: 1 / 2;
+  display: grid;
+  grid-template-rows: 1fr 90px;
 
   h1 {
     width: 100%;
     font-size: 1.2rem !important;
-    font-weight: lighter;
+    padding: 1rem;
+    background-color: var(--peach);
+    align-self: center;
+    color: var(--steelblue);
   }
 `;
 
+const SearchStyled = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  background-color: var(--peach);
+  margin: 5px auto;
+`;
+
```

Comment:
Since you only use this div for styling, would it be possible to add these styles to your Searchbar component?

Reply:
it didn't work at the component until now, now i made a note for myself and will give it a new try at the issue "complete styling make-over" scheduled probably next week